packages/beacon-node/src/sync/unknownBlock.ts

```diff
@@ -244,6 +246,8 @@ export class UnknownBlockSync {
       else {
         this.logger.error("Unknown error processing block from unknown parent sync", errorData, res.err);
         pendingBlock.status = PendingBlockStatus.pending;
+        // This potentially an orphaned block
+        this.metrics?.syncUnknownBlock.orphanedBlocks.inc();
```

Comment:
Why is this a potentially orphaned block? Orphaned block generally means a block that is included in the chain at time T1, but re-org out at time T2 > T1. You can only determine with certainty the "orphan" status at finalization or probabilistically before.

Reply:
Yah I suppose [Line 227](https://github.com/ChainSafe/lodestar/pull/5309/files#diff-593e2f511966ad9d8cb737688f30aadb67c896c68766d1b09a33cf1e251f7860R227) covers the situation where a block has an unknown parent